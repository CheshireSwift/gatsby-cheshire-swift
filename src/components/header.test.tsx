import * as React from 'react';

import { render } from 'enzyme';

import Header from './header';

describe('the header', () => {
  // it('matches the snapshot', () => {

  //   expect(render(<Header siteTitle="blah" />)).toMatchSnapshot();

  // });

  it('displays the title', () => {
    const title = 'Test Site Title Yay';

    const component = render(<Header siteTitle={title} color="white" />);

    expect(component.text()).toContain(title);
  });

  it('links to the homepage', () => {
    const component = render(<Header siteTitle="blah" color="white" />);

    expect(component).toContainSelector('a[href="/"]');
  });
});
