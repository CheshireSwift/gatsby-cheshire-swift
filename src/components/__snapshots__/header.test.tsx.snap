// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the header matches the snapshot 1`] = `
<div
  style="background:rebeccapurple;margin-bottom:1.45rem;"
>
  <div
    style="margin:0 auto;max-width:960px;padding:1.45rem 1.0875rem;"
  >
    <h1
      style="margin:0;"
    >
      <a
        href="/"
      >
        blah
      </a>
    </h1>
  </div>
</div>
`;
